phpmyfaq/admin/assets/scss/_global.scss

```
@@ -1,117 +1,365 @@
 //
-// Global definitions
+// Global admin styles
+//
```

Stylelint (17.11.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)